lib/trie/proof_test.go

```diff
@@ -0,0 +1,143 @@
+package trie
+
+import (
+	"fmt"
+	"io/ioutil"
+	"math/rand"
+	"os"
+	"testing"
+
```

Comment:
Remove this space. All standard packages should be group together.

Reply:
Done